qiskit/algorithms/factorizers/shor.py

```diff
@@ -57,10 +57,9 @@ class Shor:
     """
 
     @deprecate_function(
-        """The Shor class is deprecated as of Qiskit Terra 0.22.0 and will be removed
```

Comment:
All this diff does is convert from a multiline string to string concatenation. We can't handle `\n` in `add_deprecation_to_docstring()` (as a simplification)